R/hooks.R

```diff
@@ -1,9 +1,16 @@
 .onLoad <- function(libname, pkgname) {
   register_all_s3_methods() # dynamically registers non-imported pkgs (tidyverse) # nocov
-  assert_that(is.string(find.package("vol2birdR", verbose=F, quiet=T)),msg="package vol2birdR not found. See https://adokter.github.io/vol2birdR/ for install instructions")
+  assertthat::assert_that(assertthat::is.string(find.package("vol2birdR", verbose=F, quiet=T)),msg="package vol2birdR not found. See https://adokter.github.io/vol2birdR/ for install instructions")
 }
```

Comment:
I think this will always error out because if the package isn't found a char(0) is returned which doesn't pass the assertion.

```
> assertthat::assert_that(is.string((find.package("foobar", verbose=F, quiet=T))))
Error: (find.package("foobar", verbose = F, quiet = T)) is not a string (a length one character vector).
```

Reply:
Do I assume correctly that this PR did not break this line of code but that is was broken already? I leave it to you on how to best fix it. 😄

Reply:
Yes, you're right, it was broken before this PR.   WIll implement the change suggested above.